engine/pkg/cmd/master/master.go

```diff
@@ -74,6 +74,9 @@ func (o *options) addFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringSliceVar(&o.masterConfig.BusinessMetaConf.Endpoints, "business-meta-endpoints", o.masterConfig.BusinessMetaConf.Endpoints, "business metastore endpoint")
 	cmd.Flags().StringVar(&o.masterConfig.BusinessMetaConf.StoreType, "business-meta-store-type", o.masterConfig.BusinessMetaConf.StoreType, "business metastore store type")
+	cmd.Flags().StringVar(&o.masterConfig.BusinessMetaConf.Auth.User, "business-meta-user", o.masterConfig.BusinessMetaConf.Auth.User, "business metastore user")
+	cmd.Flags().StringVar(&o.masterConfig.BusinessMetaConf.Auth.Passwd, "business-meta-password", o.masterConfig.BusinessMetaConf.Auth.Passwd, "business metastore password")
+	cmd.Flags().StringVar(&o.masterConfig.BusinessMetaConf.Schema, "business-meta-schema", o.masterConfig.BusinessMetaConf.Schema, `schema name for business meta`)
```

Comment:
Please add comments about the usage of `frame-meta-schema` and `business-meta-schema`, including
- They stay unchanged for **one dataflow engine cluster**
- They must be different between **any two dataflow engine clusters**
- And the naming rule.

Reply:
BTW, it's better to use `framework-xxx` rather than `frame-xxx`. In computer, frame and framework often have different meanings.

Reply:
Fix it.